.env.example

```
@@ -0,0 +1,39 @@
+# ===========================================
+# ProjectVG API Server - Environment Variables Template
+# ===========================================
+# Copy this file to .env and fill in your actual values
+
+# Database Configuration
+DB_CONNECTION_STRING=Server=host.docker.internal,1433;Database=ProjectVG;User Id=sa;Password=YOUR_DB_PASSWORD;TrustServerCertificate=true;MultipleActiveResultSets=true
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)